frontend/frames/staffframe.py

```diff
@@ -85,7 +85,7 @@ def insert_staff(self, base, staff):
         self.tables_tidying()
 
     def on_listbox_select(self, event):
-        if not isinstance(self.table.selection(), str):
+        if isinstance(self.table.selection(), str):
```

Comment:
Is there any possiblity, that this `self.table.selection()` will return anything other then `tuple`?

Docs says that it wiil return selected items. That's usually mean some kind of collection. I'm getting here `tuple`.

Also what should that event callback do?

Reply:
Event removes selection from table with staff, when user selects a specialist in listbox with 'unused staff'.
You're right, my proposition was wrong. I've just pushed right fix.